### DISCUSSION TITLE
파티클 관련 문제 문의드립니다.

### DISCUSSION THREAD

**Discussion by pda017** [naverz/zepeto-studio-kor#328] [Issue Report (Legacy)]
![image](https://user-images.githubusercontent.com/77001589/152678811-b2f179dd-ed7b-4add-b0c5-6f574d853436.png)
![image](https://user-images.githubusercontent.com/77001589/152678820-e15194a2-07fb-498f-ba26-a04a32ae02f5.png)
아이템에 파티클을 추가하고 시뮬레이션 속성을 월드로 하면 기본 체형에서는 제대로 손에서 나오는데 체형이 봐뀌면
파티클 GameObject위치와 실제 파티클이 나오는 지점이 달라져요. 마치 이전 체형의 위치를 계속 쓰고 있는듯 보여요.
파티클 시뮬레이션 속성을 World로 쓸수 없을까요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 파티클의 시뮬레이션 속성을 world로 설정할 수 있고 정상 동작을 하고 있습니다.
알려주신 스크린샷으로는 구분이 어려운데 관련 현상이 계속 나타나는지 확인 부탁드리며,
문제 수정이 필요할 경우 말씀 주세요~
